plugins/in_disk/in_disk.c

```diff
@@ -214,36 +214,25 @@ static int configure(struct flb_in_disk_config *disk_config,
                      struct flb_input_instance *in)
 {
     (void) *in;
-    const char *pval = NULL;
     int entry = 0;
     int i;
+    int ret;
 
-    /* interval settings */
-    pval = flb_input_get_property("interval_sec", in);
-    if (pval != NULL && atoi(pval) >= 0) {
-        disk_config->interval_sec = atoi(pval);
-    }
-    else {
-        disk_config->interval_sec = DEFAULT_INTERVAL_SEC;
-    }
-
-    pval = flb_input_get_property("interval_nsec", in);
-    if (pval != NULL && atoi(pval) >= 0) {
-        disk_config->interval_nsec = atoi(pval);
-    }
-    else {
-        disk_config->interval_nsec = DEFAULT_INTERVAL_NSEC;
+    /* Load the config map */
+    ret = flb_input_config_map_set(in, (void *)disk_config);
+    if (ret == -1) {
+        return -1;
```

Comment:
Could you output log ?
If `flb_input_config_map_set` failed, fluent-bit will exit without error logs.